backend/src/mirrors_qa_backend/schemas.py

```diff
@@ -13,6 +13,9 @@ class BaseModel(pydantic.BaseModel):
     model_config = ConfigDict(use_enum_values=True, from_attributes=True)
 
 
+ISO3166AlphaCode = Annotated[str, Field(min_length=2, max_length=2)]
```

Comment:
There is an ISO-3166-alpha3 code as well s…
```suggestion
ISO3166Alpha2Code = Annotated[str, Field(min_length=2, max_length=2)]
```